pkg/quota/image/imagestream_evaluator_test.go

```diff
@@ -79,10 +81,17 @@ func TestImageStreamEvaluatorUsageStats(t *testing.T) {
 			expectedISCount: 1,
 		},
 	} {
-		fakeClient := &testclient.Fake{}
-		fakeClient.AddReactor("list", "imagestreams", imagetest.GetFakeImageStreamListHandler(t, tc.iss...))
-
-		evaluator := NewImageStreamEvaluator(fakeClient)
+		isInformer := cache.NewSharedIndexInformer(
```

Comment:
This is just a straight counting evaluator, right?  If so, I'm not sure you need a test at all.  The counting is checked upstream.